recipe/meta.yaml

```diff
@@ -27,10 +24,9 @@ requirements:
     - pip
     - wheel
     - setuptools
-    - numpy 
-    - setuptools-scm 7.0.4
```

Comment:
Should we add this back because of [this](https://github.com/dask/fastparquet/blob/2023.8.0/setup.py#L58)?